tests/unit/test_prometheus_metrics.py

```
@@ -1,621 +1,421 @@
+
+@pytest.fixture
+def sample_aroi_data():
+    return _sample_aroi_data
+
+
+@pytest.fixture
+def sample_dns_metadata():
+    return _sample_dns_metadata
+
+
+def _generate(
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `_generate`

(ANN202)